server/resourcemanager/testutil/operations.go

```diff
@@ -1,115 +1,46 @@
 package testutil
 
 import (
-	"encoding/json"
-	"io"
 	"net/http"
+	"net/http/httptest"
 	"testing"
-
-	"github.com/stretchr/testify/require"
 )
 
+type Operation string
+
+type operationTester struct {
```

Comment:
Folks, here is a polemic refactor that I would like to talk about with you later.

On one side, adding the operationTester as an interface is suitable for defining polymorphism and type-checking on golang. Still, for another, it generated tons of boilerplate code on our tests, making it hard to understand at first glance. 

One example was defining `postAssert` for each operation when we used it only 10% of the time. Seeing that operation was stateless, and I changed it for structs (in a js-way, you can kill me later 😛 ), omitting unused methods and reducing some declarations, like the `name` function.

Does it make sense to you? If needed, I can rollback this change and only add the pagination tests.